src/models/activityReport.js

```diff
@@ -11,12 +11,30 @@ function formatDate(fieldName) {
   return null;
 }
 
+/**
+ * Helper function called by model hooks.
+ * Updates current model instance's calculatedStatus field.
+ * Background: calculatedStatus is updated to 'submitted', 'needs_review', and 'approved'
+ * based on hooks on the ActivityReportApprovers. Before submission though,
+ * we want calculatedStatus to function like submissionStatus so developers
+ * only have to check calculatedStatus to determine overall report status.
+ * @param {*} report - current model instance
+ */
+function copyStatus(report) {
+  const { submissionStatus } = report;
+  if (submissionStatus === REPORT_STATUSES.DRAFT
+    || submissionStatus === REPORT_STATUSES.DELETED) {
+    // eslint-disable-next-line no-param-reassign
+    report.calculatedStatus = submissionStatus;
+  }
+}
+
 export default (sequelize, DataTypes) => {
   class ActivityReport extends Model {
     static associate(models) {
       ActivityReport.belongsTo(models.User, { foreignKey: 'userId', as: 'author' });
       ActivityReport.belongsTo(models.User, { foreignKey: 'lastUpdatedById', as: 'lastUpdatedBy' });
-      ActivityReport.belongsTo(models.User, { foreignKey: 'approvingManagerId', as: 'approvingManager' });
+      ActivityReport.belongsTo(models.User, { foreignKey: 'oldApprovingManagerId', as: 'oldApprovingManager' });
```

Comment:
Are we keeping the old approving manager id or are we going to run a script to move them over to the new field?

Reply:
There's a script in this PR to create ActivityReportApprover records for activity reports that existed prior to the switch to multiple approvers. It's at src/tools/transitionToMultApprovers.js. But the `oldApprovingManagerId` col will continue to hold this data for historical purposes. I didn't really think about just removing it. I suppose that's an option.

Reply:
I've now thought about removing it- and I think dropping `oldApprovingManagerId` and `oldManagerNotes` would be great. But I think it should be done in a subsequent PR a little bit later on.

Reply:
https://ocio-jira.acf.hhs.gov/browse/TTAHUB-270 for clean up work